website/pages/en/querying/graphql-api.mdx

```diff
@@ -2,7 +2,7 @@
 title: GraphQL API
 ---
 
-This guide explains the GraphQL Query API that is used for the Graph Protocol.
+This guide explains the GraphQL Query API that is used for The Graph Protocol.
```

Comment:
```suggestion
This guide explains the GraphQL Query API that is used for The Graph.
```
Source: https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#46a17f2b3fb14d6a963a6064d8efc68a